### PR TITLE
Css fix

### PR DESCRIPTION
**Pull request by FiF0o**
added css fix for spy nav broken on mobile,
clean copy and styling to make it more consistent,
clean html.

### CODE REVIEW

**Comment by panosvoudouris** [comment]
There are merge conflicts.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,42 @@
-.idea
```
Comment (panosvoudouris) [inline .gitignore]: Overall I don't think the .gitignore changes are needed, the current file is fine as is. The rest of the PR looks ok to me.

**Comment by FiF0o** [comment]
Just changed the .gitignore file and merged previous PR to be synced.
Should be ready to be merged.

**Comment by yannislionis** [comment]
Having fun during the weekend both of you?

**Pull request merged**